src/utils/git.ts

```diff
@@ -0,0 +1,36 @@
+import { exec } from 'child_process'
+import { consola } from 'consola'
+
+export function getIsGitInstalled(): Promise<boolean> {
+  return new Promise((resolve) => {
+    exec('git --version', (error, stdout, stderr) => {
+      if (error) {
+        console.error('Git is not installed.')
+        resolve(false)
+        return
+      }
+
+      if (stderr) {
+        console.error(`Git check failed. Additional info: ${stderr}`)
+        resolve(false)
+        return
+      }
+
+      console.info(`Git is installed: ${stdout}`)
```

Comment:
```suggestion
        consola.error('Git is not installed.')
        resolve(false)
        return
      }

      if (stderr) {
        consola.error(`Git check failed. Additional info: ${stderr}`)
        resolve(false)
        return
      }

      consola.info(`Git is installed: ${stdout}`)
```
Ignore this, if it is just temporary development logging